PyViCare/PyViCareRoomSensor.py

```diff
@@ -4,6 +4,10 @@
 
 class RoomSensor(Device):
 
+    @handleNotSupported
+    def getSerial(self):
+        return self.service.getProperty("device.sensors.temperature")["deviceId"]
```

Comment:
Accessing `device.sensors.temperature` does work for temp sensors and TRVs (see test data). This is a workaround as long as Viessmann does not provide `device.serial` on zigged devices (see https://community.viessmann.de/t5/Feedback-API/Serial-Number-an-Gateways/td-p/410253)